src/renderer/contexts/GlobalNodeState.tsx

```diff
@@ -102,6 +103,11 @@ interface Global {
         inputId: InputId,
         inputData: InputData
     ) => readonly [T | undefined, (data: T) => void, () => void];
+    useInputSize: (
+        id: string,
+        inputId: InputId,
+        inputSize?: InputSize
```

Comment:
I don't like this. The parameter isn't optional, it can just be undefined.
```suggestion
        inputSize: InputSize | undefined
```